source/apiVolontaria/apiVolontaria/tests/tests_view_Users.py

```diff
@@ -26,6 +29,46 @@ def setUp(self):
         self.admin.set_password('Test123!')
         self.admin.save()
 
+    def test_create_new_user_without_service_email(self):
+        """
+        Ensure we can create a new user if we have the permission.
```

Comment:
The description is not wrong, but can be more precise : `Ensure we can create a new user if we have the permission even if the email service is not activated`